internal/fileutils/fileutils.go

```diff
@@ -498,6 +498,12 @@ func MoveAllFilesRecursively(fromPath, toPath string, cb MoveAllFilesCallback) e
 				return errs.Wrap(err, "os.Remove %s failed", subFromPath)
 			}
 		} else {
+			// If the subToPath file exists, we remove it first - to ensure compatibility between in platforms:
+			// On Windows, the renaming step can otherwise fail if subToPath is read-only (file removal is allowed)
+			if toPathExists {
+				err = os.Remove(subToPath)
+				logging.Error("Failed to remove destination file %s: %v", subToPath, err)
+			}
```

Comment:
Can we take this chance to clean up the conditional flow here a little? I'm confused why this isn't in the conditional on line 482, I'm guessing it's got something to do with it being a file, but the flow of this logic is now making it really hard to grok why the code lives where it lives.

Not asking for any major refactoring, just clean up the conditionals so it's more self-explanatory.

Reply:
Yes, the block starting on line 482 is still handling with files AND directories, just checking whether the arguments are compatible.  But, yes I think I can flatten this logic a little...